root/static/scripts/tests/release-editor/seeds/mbs-13207.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <form action="/release/add" method="POST">
```

Comment:
I'd hardcoded the action URL to `https://musicbrainz.org/release/add` so I could test the seeding manually against a server with working search, but I forgot to change it back to `/release/add` before uploading the change. 😬

Reply:
You can switch “direct search” to use the entity search field in local development setup.